internal/pkg/esboot/esutil.go

```diff
@@ -2,7 +2,7 @@
 // or more contributor license agreements. Licensed under the Elastic License;
 // you may not use this file except in compliance with the Elastic License.
 
-package esutil
+package esboot
```

Comment:
nit: Would it be worth unexporting the functionality since it is only used in the `esboot` package now?

Reply:
sure, will do.